ecf/cam/plots/jevs_href_grid2obs_ecnt_past31days_plots.ecf

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+#PBS -N jevs_href_grid2obs_ecnt_plot
+#PBS -j oe
+#PBS -S /bin/bash
+#PBS -q dev
+#PBS -A VERF-DEV
+#PBS -l walltime=00:30:00
+#PBS -l place=vscatter:exclhost,select=1:ncpus=48:mem=100GB
+#PBS -l debug=true
+
+
+export OMP_NUM_THREADS=1
+
+export evs_ver=v1.0
+export HOMEevs=/lfs/h2/emc/vpppg/noscrub/${USER}/EVS
+
+source $HOMEevs/versions/run.ver
+
+module reset
+module load envvar/$envvar_ver
+module load PrgEnv-intel/$PrgEnv_intel_ver
+module load intel/$intel_ver
+module load cray-pals/$craypals_ver
+module load libjpeg/$libjpeg_ver
+module load grib_util/$grib_util_ver
+module load prod_util/$prod_util_ver
+module load prod_envir/$prod_envir_ver
+module load wgrib2/$wgrib2_ver
+module load libpng/$libpng_ver
+module load zlib/$zlib_ver
+module load jasper/$jasper_ver
+module load netcdf/$netcdf_ver
+module load cfp/$cfp_ver
+export HPC_OPT=/apps/ops/para/libs
+module use /apps/ops/para/libs/modulefiles/compiler/intel/$intel_ver
+module load gsl/${gsl_ver}
```

Comment:
Hi, Binbin,

You need to be sure to change the statements here (lines 35 and 36 in this particular ecf) to be *prod*, not *para*.  For example, they should be:

```
export HPC_OPT=/apps/ops/prod/libs
module use /apps/ops/prod/libs/modulefiles/compiler/$intel_ver
```

Would you please make these changes in all your ecf files?  Be sure they are all prod in both the stats and the plots jobs.

Reply:
OK, I have changed from para to prod for all plotting ecf files, and updated them in the github fork  
The stats ecf are already in prod